client/src/components/LabelImg/labelimg.css

```diff
@@ -0,0 +1,38 @@
+@media screen and (max-width: 500px) {
```

Comment:
Are there any styles for these components that have the `@media` selectors that you'd want to keep for all screen sizes?  Since all of these styles are currently wrapped with the `@media`, this component will appear unstyled on desktop (or rather, for all screen widths above 500px, i.e. ipad).

Reply:
The main target for is is mobile but I agree it doesn't have to be mobile only :)
We will fix this.